docs/http/http-spans.md

```diff
@@ -117,16 +117,16 @@ sections below.
 <!-- semconv trace.http.common(full) -->
```

Comment:
I think we should remove Common attributes section now that all attributes are listed on client/server. 
We can probably take it separately as a prereq to this PR so that we don't overlook something important,

Reply:
Agreed! Let's do that upstream and I can merge it in here.

Reply:
fixed in #931